wireapp/kalium
# fix: fixed bug not showing correctly generic assets on other clients [AR-3187]

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

We recently introduced some bug when refactoring the way we handle asset messages. This caused other AR clients not to display generic asset messages sent from an AR client.

### Solutions

Change the implementation of `shouldBeDisplayed()` to not rule out assets that have no metadata information.


### Testing

#### Test Coverage (Optional)


#### How to Test
Send a generic asset message from different clients, Android and/or web, and check that they are displayed correctly in AR. 

Unit tests will be coming soon too.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
